libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/GranularPermissions/GranularGranteeGroupItem.tsx

```diff
@@ -77,8 +78,9 @@ export const GranularGranteeGroupItem: React.FC<IGranularGranteeGroupItemProps>
                 className="gd-grantee-granular-permission"
                 triggerClassName="gd-grantee-granular-permission-bubble-trigger"
                 mode={mode}
+                id={id}
```

Comment:
how is it possible to have same ID on two elements?

Reply:
Added appropriate acc. config props